readthedocs/api/v2/mixins.py

```diff
@@ -35,15 +35,16 @@ def _get_cache_tags(self):
            so any exceptions like 404 should be caught.
         """
         try:
+            tags = []
             project = self._get_project()
             version = self._get_version()
-            tags = [
-                project.slug,
-                get_cache_tag(project.slug, version.slug),
-            ]
-            if self.project_cache_tag:
+            if project:
```

Comment:
```suggestion
            # Static files won't have a version, so be defensive here.          
            if project:
```

Reply:
I have added a note about the version being optional in the docstring